tests/components/alexa/__init__.py

```diff
@@ -197,4 +208,5 @@ def assert_equal(self, namespace, name, value):
                 assert prop["value"] == value
                 return prop
 
-        assert False, "property %s:%s not in %r" % (namespace, name, self.properties)
+        assert False, "property %s:%s not in %r" % (namespace, name,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)